main.pm

```diff
@@ -196,6 +196,11 @@ sub gnomestep_is_applicable() {
     return check_var("DESKTOP", "gnome");
 }
 
+sub snapper_is_applicable() {
+    my $fs = get_var("FILESYSTEM", 'btrfs');
+    return ( $fs eq "btrfs" && get_var("HDDSIZEGB") > 10);
```

Comment:
There is one other issue here. When  HDDSIZEGB is not set worker reports undefined variable in >. Please change 
```perl
get_var("HDDSIZEGB")
```
to
```perl
get_var("HDDSIZEGB", 10)
```

I fixed this for opensuse tests already